src/hooks/useUploadProfilePictureHandler.ts

```diff
@@ -35,10 +36,9 @@ export const useUploadProfilePictureHandler = (
         file = new File([resizedImage], fileName);
       }
       const storageRef = firebase.storage().ref();
-      // TODO: add rule to forbid other users to edit a user's image
       if (user?.uid) {
         const profilePictureRef = storageRef.child(
-          `/users/${user.uid}/${file.name}`
+          `/users/${user.uid}/${uuid()}_${file.name}`
```

Comment:
Any reason to add uuid here?

Reply:
Yes, so images always have unique names so we can cache them. Imagine a user uploads an image called `daniel.png` and then wants to change it and uploads another one called the same. Because of caching, the image might not get updated.

Reply:
Understood

Reply:
Added a comment to clarify https://github.com/sparkletown/sparkle/pull/2116/commits/7d66a0f519180b544b61e50b97551e37c37baa4a